site/content/en/docs/tasks/run_jobs.md

```diff
@@ -55,32 +55,10 @@ without Kueue. However, you must consider the following differences:
 - You should include the resource requests for each Job Pod.
 
 Here is a sample Job with three Pods that just sleep for a few seconds.
-This sample is also available in [github.com/kubernetes-sigs/kueue/blob/main/examples/sample-job.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/examples/sample-job.yaml).
-
-```yaml
-# sample-job.yaml
-apiVersion: batch/v1
-kind: Job
-metadata:
-  generateName: sample-job-
-  labels:
-    kueue.x-k8s.io/queue-name: user-queue
-spec:
-  parallelism: 3
-  completions: 3
-  suspend: true
-  template:
-    spec:
-      containers:
-      - name: dummy-job
-        image: gcr.io/k8s-staging-perf-tests/sleep:latest
-        args: ["30s"]
-        resources:
-          requests:
-            cpu: 1
-            memory: "200Mi"
-      restartPolicy: Never
-```
+This sample is also available
```

Comment:
The link does not work currently due to the same reason as above. I would be in favor of removing this sentence and the link since we have the include below anyway.